Client/Frontend/Settings/LoginDetailViewController.swift

```diff
@@ -178,4 +250,55 @@ extension LoginDetailViewController {
         tableView.editing = false
         navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .Edit, target: self, action: "SELedit")
     }
+
+    func SELwillShowMenuController() {
+        NSNotificationCenter.defaultCenter().removeObserver(self, name: UIMenuControllerWillShowMenuNotification, object: nil)
+
+        let menuController = UIMenuController.sharedMenuController()
+        guard let cell = menuControllerCell,
+              let textSize = cell.descriptionTextSize else {
+            return
+        }
+
+        // Hide the original menu controller so we can customize it's location and re-show
+        menuController.setMenuVisible(false, animated: false)
```

Comment:
Isn't `willShow` called before the view is shown? Why do we have to hide it here?